### ISSUE TITLE
Apple Mail in iOS 10 incorrectly scales email with fixed width and max-width:100%

### ISSUE THREAD

**Issue by hteumeuleu** [hteumeuleu/email-bugs#27]
In iOS 10 (and the first beta of iOS 11 as well), Apple Mail incorrectly scales email that contains a fixed width larger than the device viewport and a max-width of 100% (like `width:800px; max-width:100%;`). 

[Here's an example](https://cdn.rawgit.com/hteumeuleu/e264e9054648aa02b5c0d6bc6011c38c/raw/b569d97130242b3cc1089bd57552efa9915e52ac/iOS10-maxwidth.html) (and [here's the code](https://gist.github.com/hteumeuleu/e264e9054648aa02b5c0d6bc6011c38c)).

[![Screenshot of the bug](http://i.imgur.com/8h2nv34m.png)](http://i.imgur.com/8h2nv34.png)

Having the `<meta name="x-apple-disable-message-reformatting" />` tag doesn't make any difference. Inverting the `width` and `max-width` values to avoid the definition of a fixed width solves the problem (like `width:100%; max-width:800px;`).

[![Screenshot of the solution](http://i.imgur.com/HxaqP57m.png)](http://i.imgur.com/HxaqP57.png)

**Comment by revelt** [comment]
That's basically how I've been coding — `width` at 100%, if you need to set `max-width` do it in `px`, but drop some Outlook conditional rigid tables outside to contain it.

+1 for sharing the `<meta name="x-apple-disable-message-reformatting" />`. This one is new to me.

**Comment by frystpingvin** [comment]
Yeah, thanks for the `<meta name="x-apple-disable-message-reformatting" />`. It helped me solve another issue!

**Comment by mealyjc** [comment]
Anyone still having issues with this? Still can't seem to get the email to rescale for iPhone - have tried all suggestions above and others
